sparksteps/__main__.py

```diff
@@ -185,6 +187,11 @@ def warn_and_override(args, argument, overrides, force_override=False):
     # Alias 'dynamic-pricing' -> 'dynamic-pricing-task'.
     args = warn_and_override(args, 'dynamic_pricing', ['dynamic_pricing_task'])
 
+    # Perform sanitization on any CLI arguments
+    if args['s3_path'] and args['s3_path'].startswith('/'):
+        raise ValueError(
+            'Provided value for s3-path "{S3_PATH}" cannot have leading "/" character.'.format(args.s3_path))
```

Comment:
there was a decision here whether to raise or remove and continue, decided to be explicit and fail fast

Reply:
the reason why this is unsupported is due to the fact that:

```
>>> os.path.join('foo', '/bar')
'/bar'
>>> os.path.join('foo', 'bar/baz')
'foo/bar/baz'
```

Reply:
The default value is `''` and that should be OK as well:

```
>>> os.path.join('foo', '', 'bar/baz')
'foo/bar/baz'
```

Reply:
I think `args.s3_path` might raise an exception and should be args['s3_path']